newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -236,7 +236,7 @@ export const AssetStoreStateProvider = ({
         isLoading.current = false;
       })();
     },
-    [assetShortHeadersById, isLoading]
+    [isLoading]
```

Comment:
This was not necessary?

Reply:
Instead of:
- calling this function every time the NewObjectDialog opens (with an effect)
- handling in the context if there are already elements, to decide if the call happens

I think it's cleaner to:
- call this function by the context on first load
- call manually when needed (retry / switch environment)

I didn't notice any issue, but now that I think about it, there may be a slight delay if you open the asset store and the context hasn't started fetching the assets yet (quite unique case)
I'll do more testing